src/Controls/src/Core/IMarkupExtension.cs

```diff
@@ -18,4 +18,14 @@ public interface IMarkupExtension
 	public sealed class AcceptEmptyServiceProviderAttribute : Attribute
 	{
 	}
+
+	[AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
```

Comment:
Should we allow multiple attributes since the attribute takes an array of types? I can't see any usage of multiple attributes and all the places in `SetPropertiesVisitor` where you access the attribute on a type seem to assume there's at most one.

Reply:
Allow Multiple should be false. fixing